containers/python/rpc.py

```diff
@@ -332,7 +332,10 @@ def run(self, metric_conn):
                 sys.stderr.flush()
 
     def send_container_metadata(self, socket):
-        socket.send("", zmq.SNDMORE)
+        try:
+            socket.send("", zmq.SNDMORE)
+        except:
+            socket.send("".encode('utf-8'), zmq.SNDMORE)
```

Comment:
Is this a Python 2 vs 3 thing? If so, instead of trying one way and catching an exception, let's explicitly check for sys.version like you do in the python deployer and then just use the right version. Throwing and catching an exception can be expensive and the code as-is has unclear semantics.